tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -165,7 +183,16 @@ class CompletionOverrideSuite extends BaseCompletionSuite {
        |override def clone(): Object
        |override def finalize(): Unit
        |""".stripMargin,
-    includeDetail = false
+    includeDetail = false,
+    compat = Map(
+      "3" ->
+        """|override def aaa: Int
+           |override def bbb: Int
+           |override def equals(x$0: Any): Boolean
```

Comment:
This might change after rebasing.

Reply:
For Java symbols, it works :+1: https://github.com/scalameta/metals/pull/3897/commits/a484aec9e5a0a548ecc069359496a43cd842f6cd